src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -862,55 +874,40 @@ def apply_ldlc_treatment_ramp(
         treatment_change_eligible = treatment_change_eligible_young.union(
             treatment_change_eligible_old
         )
-
-        changed_prescription_last_time = overcome_prescription_inertia.intersection(
-            treatment_change_eligible
-        )
-        ldlc_prescription_inertia_propensity.loc[
-            changed_prescription_last_time
-        ] = self.randomness.get_draw(
-            changed_prescription_last_time, additional_key="dynamic_ldlc_inertia_propensity"
+        high_ldlc = measured_ldlc[measured_ldlc >= data_values.LDLC_THRESHOLD.HIGH].index
+        mask_history_ihd_or_hf = (
+            pop_visitors[models.ISCHEMIC_HEART_DISEASE_AND_HEART_FAILURE_MODEL_NAME]
+            != models.ISCHEMIC_HEART_DISEASE_AND_HEART_FAILURE_SUSCEPTIBLE_STATE_NAME
         )
-        updated_overcome_prescription_inertia = pop_visitors[
-            ldlc_prescription_inertia_propensity > data_values.LDLC_THERAPEUTIC_INERTIA
-        ].index
-
-        newly_prescribed_young = (
-            updated_overcome_prescription_inertia.difference(currently_medicated)
-            .difference(low_ascvd)
-            .difference(low_ldlc)
-            .difference(old_pop)
+        mask_history_is = (
+            pop_visitors[models.ISCHEMIC_STROKE_MODEL_NAME]
+            != models.ISCHEMIC_STROKE_SUSCEPTIBLE_STATE_NAME
         )
+        history_ihd_hf_or_is = pop_visitors[mask_history_ihd_or_hf | mask_history_is].index
 
         # [Treatment ramp ID D] Simulants who overcome therapeutic inertia, have
         # elevated LDLC, are not currently medicated, have elevated ASCVD, and
         # have a history of MI or IS
-        to_prescribe_d = newly_prescribed_young.intersection(history_mi_or_is)
+        to_prescribe_d = newly_prescribed_young.intersection(history_ihd_hf_or_is)
         # [Treatment ramp ID E] Simulants who overcome therapeutic inertia, have
         # elevated LDLC, are not currently medicated, have elevated ASCVD, have
         # no history of MI or IS, and who have high LDLC or ASCVD
         to_prescribe_e = newly_prescribed_young.difference(to_prescribe_d).intersection(
             high_ascvd.union(high_ldlc)
         )
         # [Treatment ramp ID F] Simulants who overcome therapeutic inertia, have
-        # elevated LDLC, are not currently medicated, have elevated ASCVD, have
-        # no history of MI or IS, but who do NOT have high LDLC or ASCVD
-        to_prescribe_f = newly_prescribed_young.difference(to_prescribe_d).difference(
+        # elevated LDLC, are not currently medicated, and are either (1) old or
```

Comment:
Nit: is it worth explicitly saying "<75" here? If we think the value might change, it probably shouldn't be hard-coded in ln 854